index.html

```diff
@@ -378,20 +379,45 @@
     which can originate either from an <a>NFC tag</a> or an <a>NFC peer</a>.
   </p>
   <p>
-    An <dfn>NDEF record</dfn> is a part of an <a>NDEF message</a> that has a
-    single associated type information to its payload. It contains a
-    Type Name Format (TNF) field, the payload size, the payload type, an
-    optional identifier which is a URL, and a payload of maximum size of
-    2^32-1 bytes.
-    The NFC Forum has standardized a small set of useful data types for
-    use in <a>NDEF record</a>s, for instance text, URL, and binary data such as
-    media. In addition, there are record types designed for more complex
-    interactions, such as Smart Poster, and handover records.
+    An <dfn>NDEF record</dfn> is a part of an <a>NDEF message</a>. Each record
+    is a binary structure that contains a data payload, as well as associated
+    type information. In addition to this, it includes information about how
+    the data is structured, like payload size, whether the data is chunked over
+    multiple records etc.
   </p>
   <p>
-    The <dfn>TNF</dfn> (Type Name Format) field of the <a>NDEF record</a> can
-    take binary values denoting the following <a>NDEF record</a> payload
-    types:
+    A generic record looks like the following:
+    <ndef-record class="ndef"
+      header="*,*,*,*,*,*"
+      content="*,PAYLOAD LENGTH - 1 (SR) to 4 bytes,ID LENGTH (optional),TYPE (optional),ID (optional),PAYLOAD (optional)">
+    </ndef-record>
+  </p>
+  <p>
+    Only the first three bytes (lines in figure) are mandatory. First the header,
+    followed by the type length and payload length, which may both be zero.
+  </p>
+  <p>
+    Bit 7 of the header (MB, message begin) indicates whether this is the beginning
```

Comment:
Why not just split definitions of each bits into separate paragraphs? BTW, these could be `dfn` as which had been referenced in somewhere of the spec.